report_alternative_layout/report/report_template.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template id="external_layout_standard" inherit_id="web.external_layout_standard">
+        <!-- Hide company address in the header if alternative layout is selected -->
+        <xpath expr="//div[@name='company_address']/ul" position="attributes">
+            <attribute
+                name="t-if"
+            >not env.context.get('apply_alternative_layout')</attribute>
+        </xpath>
+        <!-- Show company address in the body if alternative layout is selected -->
+        <xpath expr="//t[@t-call='web.address_layout']" position="attributes">
+            <attribute
+                name="t-if"
+            >not env.context.get('apply_alternative_layout')</attribute>
+        </xpath>
+        <xpath expr="//t[@t-call='web.address_layout']" position="after">
+            <t t-else="">
+                <div class="row" name="address_layout">
+                    <div class="col-7" name="partner_address">
+                        <t
+                            t-set="partner_name"
+                            t-value="o.env['ir.actions.report']._get_partner(xmlid, o.partner_id)"
+                        />
+                        <div
+                            t-esc="partner_name"
+                            t-options='{"widget": "contact", "fields": ["address", "name"], "no_marker": true}'
+                        />
+                    </div>
```

Comment:
```suggestion
                    </div>
                                <div t-if="o.partner_id.vat">
                                    <t>Tax ID</t>: <span t-field="o.partner_id.vat"/>
                                </div>
```
I found that partner's Tax ID is not printing.

Reply:
Sorry, Forget this comment. It is not always necessary to show the partner's VAT in Japan.